exdir/core/exdir_file.py

```diff
@@ -220,6 +220,12 @@ def __getitem__(self, name):
             return self
         return super(File, self).__getitem__(path)
 
+    def __setitem__(self, name, value):
```

Comment:
Is this related to links or is it just something we have left out from before? Seems like this is something that has been missing on File regardless of links?